### ISSUE TITLE
526: Remove old feature flags

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#38842]
## Description

Form 526 still has a few feature flags in the code that have been at 100%, or already remove, and need to be cleaned up

- `request_locked_pdf_password`
- `evss_upload_limit_150mb`

## Tasks

- [ ] Remove feature flag code in 526
- [ ] Update tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing

**Comment by Mottie** [comment]
cc: @saderagsdale

**Comment by jerekshoe** [comment]
Once this is done, we will need another ticket to remove these feature flags from `vets-api` as well

**Comment by michelpmcdonald** [comment]
Jared Cunha is current working this issue.  He is having some kind of github  permission issue that is preventing assigning this issue to him

**Comment by Mottie** [comment]
It looks like this work was done for 526, but it still remains for COE, platform code and vets-api features.

**Comment by Mottie** [comment]
Oops, a [PR for some of this work is still open](https://github.com/department-of-veterans-affairs/vets-website/pull/21120)
